packages/dashboard-v2/src/components/forms/AddSkylinkToAPIKeyForm.js

```diff
@@ -0,0 +1,73 @@
+import * as Yup from "yup";
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+
+import accountsService from "../../services/accountsService";
+
+import { Button } from "../Button";
+import { TextField } from "../Form/TextField";
+import { CircledProgressIcon, PlusIcon } from "../Icons";
+
+const newSkylinkSchema = Yup.object().shape({
+  skylink: Yup.string().required("Provide a valid Skylink"), // TODO: Comprehensive Skylink validation
```

Comment:
I think a length check should suffice so it's probably easy to do here in this PR.  
Probably you also want to take into account the `sia://` prefix or the portal url prefixes.

Reply:
I found `parseSkylink` in `skynet-js` :) It's gonna help both with validation and accepting `sia://` and portal URLs as skylinks to be pinned: 9dd4b8eb